README.md

```diff
@@ -153,13 +153,13 @@ Processing references: 539
 
 The output is a table showing the thing that was measured, its numerical value, and a rough indication of which values might be a cause for concern. In all cases, only objects that are reachable from references are included (i.e., not unreachable objects, nor objects that are reachable only from the reflogs).
 
-The "Overall repository size" section includes repository-wide statistics about distinct objects, not including repetition. "Total size" is the sum of the sizes of the corresponding objects in their uncompressed form, measured in bytes.
+The "Overall repository size" section includes repository-wide statistics about distinct objects, not including repetition. "Total size" is the sum of the sizes of the corresponding objects in their uncompressed form, measured in bytes. When the uncompressed size is much larger than the on-disk size, that is an indication that things are compressing very well or that there are large blobs or large trees that are getting modified incrementally many times. The overall uncompressed size of all objects is a good indication of how expensive commands like `git gc --aggressive` (and `git repack [-f|-F]` and `git pack-objects --no-reuse-delta`), `git fsck`, and `git log [-G|-S]` will likely be.
 
 The "Biggest objects" section provides information about the biggest single objects of each type, anywhere in the history.
 
 In the "History structure" section, "maximum history depth" is the longest chain of commits in the history, and "maximum tag depth" reports the longest chain of annotated tags that point at other annotated tags.
 
-The "Biggest checkouts" section is about the sizes of commits as checked out into a working copy. "Maximum path depth" is the largest number of path components for files in the working copy, and "maximum path length" is the longest path in terms of bytes. "Total size of files" is the sum of all file sizes in the single biggest commit, including multiplicities if the same file appears multiple times.
+The "Biggest checkouts" section is about the sizes of commits as checked out into a working copy. "Maximum path depth" is the largest number of path components for files in the working copy, and "maximum path length" is the longest path in terms of bytes. "Total size of files" is the sum of all file sizes in the single biggest commit, including multiplicities if the same file appears multiple times. The uncompressed size of trees and commits is a good indication of how expensive reachability traversals will be, including clones and fetches and `git gc`.
```

Comment:
The "The uncompressed size ..." sentence also belongs in the section about "Overall repository size". The total sum of these sizes indicates more about the time to traverse all of history, which isn't necessarily correlated with the time to correlate a single tree.

"Total size of files" is different. It is roughly the disk space needed to check out the biggest commit in the history. It is possible that this includes multiple copies of the same blob (at different paths). "Overall repository size", by contrast, never includes multiple copies of the same exact blob.